src/include/cell.i

```diff
@@ -133,19 +161,20 @@ __wt_check_addr_validity(WT_SESSION_IMPL *session, wt_timestamp_t oldest_start_t
  *     Pack the validity window for an address.
  */
 static inline void
-__cell_pack_addr_validity(WT_SESSION_IMPL *session, uint8_t **pp, wt_timestamp_t newest_durable_ts,
-  wt_timestamp_t oldest_start_ts, uint64_t oldest_start_txn, wt_timestamp_t newest_stop_ts,
-  uint64_t newest_stop_txn)
+__cell_pack_addr_validity(WT_SESSION_IMPL *session, uint8_t **pp, wt_timestamp_t oldest_durable_ts,
+  wt_timestamp_t newest_durable_ts, wt_timestamp_t oldest_start_ts, uint64_t oldest_start_txn,
+  wt_timestamp_t newest_stop_ts, uint64_t newest_stop_txn, bool prepare)
 {
     uint8_t flags, *flagsp;
 
+    /* FIXME: Check validity of durable timestamps. */
     __wt_check_addr_validity(
       session, oldest_start_ts, oldest_start_txn, newest_stop_ts, newest_stop_txn);
 
     /* Globally visible values have no associated validity window, set a flag bit and store them. */
-    if (newest_durable_ts == WT_TS_NONE && oldest_start_ts == WT_TS_NONE &&
-      oldest_start_txn == WT_TXN_NONE && newest_stop_ts == WT_TS_MAX &&
-      newest_stop_txn == WT_TXN_MAX)
+    if (oldest_durable_ts == WT_TS_NONE && newest_durable_ts == WT_TS_MAX &&
```

Comment:
@kommiharibabu, I don't think the code here contradicts what you say.  I think this code is handling an optimization for the "base case".  The "base case" would be when the application uses no time stamps or durability (like the simplest WT test case).  In that case, all of these values will be in their "initial condition", which for oldest_durable or start_durable is WT_TS_NONE and for newest_durable or stop_durable is WT_TS_MAX.  In that simple case, we have the most compact encoding, which is that we don't need to write the second descriptor byte, because all these variables are in their initial state.

If I'm missing something or you believe that the oldest_durable_ts or durable_stop_ts base case value should not be WT_TS_MAX, let's have a discussion.

Reply:
@ddanderson, I added detailed discussion summary of how the aggregated timestamp values should be and their values in the Jira ticket. According to the summary, we should use the same default value for `start` and `stop` durable timestamps. As both of them don't consider the `WT_TS_MAX` in their calculations, we could live with `WT_TS_NONE` as the default value for both of them?